docker-entrypoint-funcs.sh

```diff
@@ -1,10 +1,16 @@
 #!/usr/bin/env bash
 
+declare -A _edbdocker_log_levels=([trace]=0, [debug]=1, [info]=2, [warning]=3)
+_edbdocker_log_level="info"
 
 edbdocker_setup_shell() {
   set -Eeo pipefail
   shopt -s dotglob inherit_errexit nullglob compat"${BASH_COMPAT=42}"
-  if [ -n "$EDGEDB_SERVER_DOCKER_TRACE" ]; then
+  if [[ -n ${EDGEDB_SERVER_DOCKER_LOG_LEVEL+x} ]]; then
```

Comment:
Wouldn't this result in `_edbdocker_log_level` being set to an empty string if I pass `env EDGEDB_SERVER_DOCKER_LOG_LEVEL=""`?

Reply:
I added a check for invalid log level values.